### PR TITLE
Updated migration 18

### PR DESCRIPTION
**Pull request by daviddyess**
**18.do.flavor-safety.sql**
- added cas_number to uk1_ingredient
- removed duplicate 'Peppermint oil' entry in insert

### CODE REVIEW

File: schema/18.do.flavor-safety.sql [hunk 1]
```diff
@@ -23,7 +23,7 @@ create table ingredient (
   updated timestamp default null,
 
   constraint pk_ingredient primary key (id),
-  constraint uk1_ingredient unique (name),
+  constraint uk1_ingredient unique (name, cas_number),
```
Comment (ayan4m1) [inline schema/18.do.flavor-safety.sql]: Are there multiple CAS for individual ingredients now, or are you just predicting the need for this?

I think we should _either_

 * Leave the constraints as they are
 * Make this change to uk1_ingredient and remove uk2_ingredient

Comment (daviddyess) [reply schema/18.do.flavor-safety.sql]: My assumptions are: there is a possibility of having a name with multiple CASRNs, but there shouldn't be any duplicate CASRNs. That's just based on our current data set and my understanding of CAS. 

If we don't change uk1 then we have to remove one of the anise oil inserts.

Comment (ayan4m1) [reply schema/18.do.flavor-safety.sql]: OK, if you don't mind just remove uk2_ingredient then, otherwise I think uk2 will still be violated.

Comment (daviddyess) [reply schema/18.do.flavor-safety.sql]: Removed uk2_ingredient

**Pull request merged**